pkg/components/contour/component_test.go

```diff
@@ -52,6 +52,15 @@ component "contour" {
 	testRenderManifest(t, configHCL)
 }
 
+func TestRenderManifestWithIngressHostsWildcard(t *testing.T) {
+	configHCL := `
+component "contour" {
+  ingress_hosts = ["*.domain.com"]
```

Comment:
Actually, what breaks the rendering? the wildcard or multiple entires?

Reply:
wildcard breaks the rendering.

multiple entries didn't throw any error, but were rendered incorrectly.